grpc-kotlin-gen/src/main/java/io/rouz/grpc/kotlin/GrpcKotlinGenerator.java

```diff
@@ -227,6 +227,9 @@ private String getJavaDoc(String comments, String prefix) {
       StringBuilder builder = new StringBuilder("/**\n")
           .append(prefix).append(" * <pre>\n");
       Arrays.stream(HtmlEscapers.htmlEscaper().escape(comments).split("\n"))
+          // Kotlin allows nested block comments, so any occurrence of '/*' would begin
+          // a nested block, breaking the generated code.
+          .map(line -> line.replace("/*", "/&#42;"))
```

Comment:
You'll need to also escape the closing sequence `*/`. In the current form, this will cause the `*/` sequence close the outer comment block and leave the rest of the comment outside the block.

I tested this with

```
// foobar
// /* nested comments */  bla bla
// hello
service Greeter {
```

I suggest adding a test (or just adding some comments to the existing test service) to ensure this works.